tools/msys2_setup_buildenv_stage2_32_64.sh

```diff
@@ -0,0 +1,211 @@
+  ${MINGW_PACKAGE_PREFIX}-freetype \
+  ${MINGW_PACKAGE_PREFIX}-lcms2 \
+  ${MINGW_PACKAGE_PREFIX}-libwebp \
+  ${MINGW_PACKAGE_PREFIX}-openjpeg2 \
+  ${MINGW_PACKAGE_PREFIX}-libimagequant \
+  ${MINGW_PACKAGE_PREFIX}-libraqm \
+  ${MINGW_PACKAGE_PREFIX}-SDL2 \
+  ${MINGW_PACKAGE_PREFIX}-clang \
+  ${MINGW_PACKAGE_PREFIX}-nsis \
+  ${MINGW_PACKAGE_PREFIX}-dfu-util
+
+  end_step $? "pacman -S --noconfirm <packages>"
+
+  run_step "Upgrading pip" "python -m pip install -U pip"
+  run_step "Installing Python clang" "python -m pip install clang"
+  run_step "Installing Python setuptools and wheel" "python -m pip install setuptools wheel"
+  run_step "Installing Python jinja2" "python -m pip install jinja2"
+
+  # Nov 2023
+  # From https://github.com/miurahr/aqtinstall Install section
+  # When you want to use it on MSYS2/Mingw64 environment, you need to set environmental variable export SETUPTOOLS_USE_DISTUTILS=stdlib,
+  # because of setuptools package on mingw wrongly raise error VC6.0 is not supported
+  export SETUPTOOLS_USE_DISTUTILS=stdlib
+
+  # this will also install dependencies and see note above re psutil issue
+  run_step "Installing Python package aqtinstall" "python -m pip install aqtinstall"
+fi
+
+if [[ $INSTALL_QT -eq 1 ]]; then
+  # if modules use syntax -m MODULE [MODULE]
+  QT_MODULES=
+  # supported versions of dependencies
+  QT_TOOLS=
+  QT_HOST=windows
+  QT_TARGET=desktop
+  QT_ARCH=
+  QT_INSTALL_DIR=qt
+
+  new_step "Searching for available Qt ${QT_VERSION} architectures"
+
+  qtarchlist="$(python -m aqt list-qt ${QT_HOST} ${QT_TARGET} --arch ${QT_VERSION})"
+
+  IFS=' ' read -ra qtarcharray <<< ${qtarchlist}
+
+  for arch in "${qtarcharray[@]}"; do
+    if [[ "${arch}" == win${MSYSTEM:(-2)}_mingw* ]];then
+      QT_ARCH=${arch}
+      break
+    fi
+  done
+
+  if [[ -z "$QT_ARCH" ]]; then
+    end_step 1 "Unable to find suitable Qt architecture in Qt ${QT_VERSION} for ${MSYSTEM}"
+  else
+    end_step 0 ""
+  fi
+
+  run_step "Installing Qt ${QT_VERSION}" \
+  "python -m aqt install-qt --outputdir ${QT_INSTALL_DIR} ${QT_HOST} ${QT_TARGET} ${QT_VERSION} ${QT_ARCH} ${QT_MODULES}"
+fi
+
+if [[ $DOWNLOAD_ARM -eq 1 ]]; then
+  # Note: the version needs to be kept in sync with EdgeTX/build-edgetx/dev/Dockerfile
+
+  DOWNLOAD_FILE="gcc-arm-none-eabi-10-2020-q4-major-win32.exe"
+
+  run_step "Downloading ARM installer" \
+  "wget -c -O ${DOWNLOAD_DIR}/${DOWNLOAD_FILE} --progress=bar:force:noscroll --no-check-certificate \
+  'https://developer.arm.com/-/media/Files/downloads/gnu-rm/10-2020q4/${DOWNLOAD_FILE}?revision=9a4bce5a-7577-4b4f-910d-4585f55d35e8&rev=9a4bce5a75774b4f910d4585f55d35e8&hash=9770A44FEA9E9CDAC0DD9A009190CC8B'"
+fi
+
+echo "This stage of setting up EdgeTX build environment has finished"
+
+if [[ $DOWNLOAD_ARM -eq 1 ]]; then
+  echo "
+  Next steps:
+  1. Exit this terminal session
+
+  2. In Windows, install the downloaded ARM GNU 11.3.1 toolchain
+
+    Launch the installer ${DOWNLOAD_FILE}
```

Comment:
By the script downloaded file and also the $[DOWNLOAD_FILE] expression lists `gcc-arm-none-eabi-10-2020-q4-major-win32.exe` on the MSYS2 terminal. Line 199 above requests to install ARM GNU 11.3.1. This means to manually download from https://developer.arm.com/downloads/-/arm-gnu-toolchain-downloads and install `arm-gnu-toolchain-11.3.rel1-mingw-w64-i686-arm-none-eabi.exe`